chapters/2.md

```diff
@@ -1,4 +1,195 @@
 # Chapter 2
 
-## Getting Started
+# Getting Started
+The last chapter focused on WHY to use Monero; in this chapter you will learn HOW to use Monero. You can master using Monero without needing to learn any of the complex cryptographic or network technical details, so that extra information is saved for later in the book. **Chapter 2 will cover all the practical skills you’ll need to get started receiving,
+storing, and spending your Moneroj**.
 
+**The first part of this chapter covers key concepts and terminology for Monero use, as general information that will apply to any wallet or software.** Toward the end of the chapter, you’ll find handy guides for carrying out these steps using the free official open-source Monero command line interface (CLI) or graphical user interface (GUI) software.
+
+## What is a Wallet?
+
+Before you obtain some Moneroj, you must plan ahead for where you will receive and store your funds. You will need a “wallet” to help you store and spend your Moneroj. Just like physical wallets for your bills, there are lots of different types of Monero wallets, and you can always move some of your money from an old wallet to a new one.
```

Comment:
`Before obtaining some Moneroj, you must have a place to receive and store your funds. Just like a physical wallet where you would store coins or bills, you will need a wallet to help store and spend your Moneroj. There are many different types of wallets, but you can always move your money from an old wallet to a different one.`